lib/matplotlib/backend_bases.py

```diff
@@ -808,7 +808,12 @@ def get_clip_path(self):
         an affine transform to apply to the path before clipping.
         """
         if self._clippath is not None:
-            return self._clippath.get_transformed_path_and_affine()
+            tpath, tr = self._clippath.get_transformed_path_and_affine()
+            if np.all(np.isfinite(tpath.vertices)):
+                return tpath, tr
+            else:
+                warnings.warn("Ill-defined clip_path detected. Returning None.")
```

Comment:
This should be _log.warning(...).

Reply:
Which will also reduce the line length of that line by 1 and thus make Travis (flake8) pass.